File: javascript-closure-angular-client/API/Client/SalesTransactionIn.js
goog.provide('API.Client.SalesTransactionIn');

/**
 * @record
 */
API.Client.SalesTransactionIn = function() {}

/**
 * @type {!API.Client.SalesHeaderIn}
 * @export
 */
API.Client.SalesTransactionIn.prototype.header;

/**
 * @type {!Array<!API.Client.SalesLinesIn>}
 * @export
 */
API.Client.SalesTransactionIn.prototype.lines;

File: dot-net-client/src/Avalara.AvaTax.Brazil.RestClient/Model/SalesTransactionIn.cs
using System.Collections.Generic;
using System.Runtime.Serialization;

namespace Avalara.AvaTax.Brazil.RestClient.Model
{
    /// <summary>
    /// SalesTransactionIn
    /// </summary>
    [DataContract]
    public class SalesTransactionIn
    {
        /// <summary>
        /// Gets or Sets Header
        /// </summary>
        [DataMember(Name = "header", EmitDefaultValue = false)]
        public SalesHeaderIn Header { get; set; }

        /// <summary>
        /// Gets or Sets Lines
        /// </summary>
        [DataMember(Name = "lines", EmitDefaultValue = false)]
        public List<SalesLinesIn> Lines { get; set; }
    }
}